### PR TITLE
Fix some favicon issues

### PR DESCRIPTION
**Pull request by felixfontein**
See discussion in #ansible-community.

### CODE REVIEW

**Comment by felixfontein** [comment]
Ok, I also resolved the last issue: that the icons weren't shown on Safari. The problem is that Safari does **not** support SVGs for icons :-( So if we use SVGs, Safari will not display them. That's why I converted the SVGs to PNGs. I included instructions on how to do the conversion with ImageMagick in the commit message; where should I properly document it? In the README, or somewhere else?

**Comment by gundalow** [comment]
Thanks for looking into this.
I guess document in the commit message and in this PRs description.

**Comment by gundalow** [comment]
Actually, we have original logos here https://www.ansible.com/logos

**Comment by felixfontein** [comment]
Thanks, I used the images from there and added instructions to the README how they are obtained and how I modified them (adjusting the SVG bounding box; rescaling the PNG versions).

File: README.rst [hunk 1]
```diff
@@ -4,3 +4,16 @@ sphinx_ansible_theme
 A reusable Ansible Sphinx Theme.
 
 Demo-site at https://sphinx-ansible-theme.readthedocs.io/en/latest/
+
+Logos in ``sphinx_ansible_theme/static/images/``
```
Comment (gundalow) [inline README.rst:8]: Great info, thanks for documenting this

**Review by gundalow — APPROVED** [review]
Thanks for digging into this.

**Comment by felixfontein** [comment]
CC @webknjaz

**Review by ssbarnea — APPROVED** [review]

**Comment by felixfontein** [comment]
@gundalow @ssbarnea thanks for reviewing and merging!

**Pull request merged**